docs/tutorial/app-feedback-program.md

```diff
@@ -0,0 +1,2 @@
+# Electron App Feedback Program
+Electron is working on building a streamlined release process and having faster releases. To help with that, we have the App Feedback Program for large-scale Electron apps to report app-specific blockers to the Electron team. We use this program to help us prioritize work and get applications upgraded to the next stable release as soon as possible. There are a few requirements we expect from participants, such as attending weekly check-in meetings. If you are interested or have questions, please send us a message at info@electronjs.org.
```

Comment:
Short, friendly, and to the point. :heart: 

Minor:

 * I'd make explicit that we'd like the reported issues to be on betas. Possible rewording: "for large-scale Electron apps to test our beta releases and report..."

 * s/app-specific blockers/app-specific issues/

 * s/weekly check-in meetings/short, online weekly check-ins/